### ISSUE TITLE
[Question] What is the main difference between OpenCvSharp for C# & For mobile?

### ISSUE THREAD

**Issue by craftindo** [Kawaian/OpenCvSharp#14]
I've seen some differences in OpenCvSharp folder.
Or, basically my question is what is needed to make OpenCvSharp work in mobile platform?

thanks

**Comment by ystallonne** [comment]
OpenCvSharp is a wrapper of OpenCV to .NET, the problem is that is is not compatible with the ARM architecture, which means it will not run on smartphones. In other words, it all needs to be "adapted" to run on ARM devices.

**Comment by DamianSuess** [comment]
I agree w/ @craftindo request. Is it possible to have the code changes documented in the Wiki or somewhere to show differences?

I would love to help contribute to this repository, however, the unfamiliarity with the areas that have been modified limits one's effectiveness.  Thank you for making this fork!
